cmd/alertmanager/main.go

```diff
@@ -96,6 +96,11 @@ var (
 			Help: "Number of configured integrations.",
 		},
 	)
+	configuredInhibitRules = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Name: "alertmanager_inhibit_rules",
```

Comment:
```suggestion
			Name: "alertmanager_inhibiton_rules",
```

The config option is `inhibit` yes, but the term used is "inhibition rules".